src/screens/Me/Me.js

```diff
@@ -51,18 +53,19 @@ class MeScreen extends React.Component<Props, State> {
   };
 
   validateWalletConnectQRCode = (uri: string) => {
-    if (uri.startsWith('wc:')) {
-      return true;
-    }
-    return false;
+    return uri.startsWith('wc:') || uri.startsWith('pillarwallet:');
   };
 
   toggleQRScanner = () => this.setState({ isScanning: !this.state.isScanning });
 
   handleQRScannerClose = () => this.setState({ isScanning: false });
 
   handleQRRead = (uri: string) => {
-    this.props.onWalletConnectSessionRequest(uri);
+    if (uri.startsWith('wc:')) {
+      this.props.onWalletConnectSessionRequest(uri);
```

Comment:
Maybe destruct from props and then excecute the destructed fns?

Reply:
Done, had to rename this one since it would shadow the imported function.